dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/relationship/RelationshipOperationParams.java

```diff
@@ -42,51 +43,97 @@
 import org.hisp.dhis.tracker.export.Order;
 
 @Getter
-@Builder(toBuilder = true)
 @AllArgsConstructor(access = AccessLevel.PRIVATE)
 public class RelationshipOperationParams {
   private TrackerType type;
```

Comment:
Can these fields now all be final 🤔 as the builder sets its own fields and on build passes them on to this classes constructor